src/App.tsx

```diff
@@ -1,61 +1,144 @@
+import { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { ToDo } from './types/type';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(id: number) {
+  return usersFromServer.find(usr => (id) === usr.id) || null;
+}
 
 export const App = () => {
+  const initTodos = todosFromServer.map(todo => ({
+    ...todo,
+    user: getUser(todo.userId),
+  }));
+
+  const [todos, setTodos] = useState<ToDo[]>(initTodos);
+  const [todo, setTodo] = useState<ToDo>(
+    {
+      id: 0,
+      title: '',
+      completed: false,
+      user: null,
+    },
+  );
+
+  const [isFilled, setIsFilled] = useState(true);
+
+  const onAdd = (newTodo: ToDo) => {
```

Comment:
this function name should be rather `addTodo` to have more descriptive name instead of `onAdd`